### DISCUSSION TITLE
Invalid <buckets> parameter: 2^3 (supported: 2^[4..16])

### DISCUSSION THREAD

**Discussion by grayfallstown** [madMAx43v3r/chia-plotter#172] [General]
Is there a technical reason for not allowing 2^3 buckets?
My system has 64GB ram, so using a ramdisk is not an option for me and since parallel plotting is basically off the table with this plotter I want to make the most of my available ram.

**Comment by madMAx43v3r** [comment]
that's 8 buckets... that'll just thrash your RAM for no good.. the sorting will be slower too, I use bucket quick-sort in-memory, it gets slower with less buckets at some point
